transforms/code/license_copyright_removal/python/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM docker.io/python:3.11.9-slim-bullseye
+
+RUN pip install --upgrade pip 
+
+# install pytest
+RUN pip install --no-cache-dir pytest
+
+# Create a user and use it to run the transform
+RUN useradd -ms /bin/bash dpk
+USER dpk
+WORKDIR /home/dpk
+
+# Copy and install data processing libraries 
+# These are expected to be placed in the docker context before this is run (see the make image).
+COPY --chown=dpk:root data-processing-lib-python/ data-processing-lib-python/
+RUN cd data-processing-lib-python && pip install --no-cache-dir -e .
+
+# END OF STEPS destined for a data-prep-kit base image 
+
+COPY --chown=dpk:root src/ src/
+COPY --chown=dpk:root pyproject.toml pyproject.toml 
+RUN pip install --no-cache-dir -e .
+
+#COPY requirements.txt requirements.txt
```

Comment:
@ykalathiya i think, this needs to be uncommented. I see a docker build failure due to scancode module missing...
https://github.com/IBM/data-prep-kit/actions/runs/9560117498/job/26352464226?pr=239